flow/templates/lsf.sh

```diff
@@ -2,6 +2,7 @@
 {% block header %}
 #!/bin/bash
 #BSUB -J {{ id }}
+#BSUB -M {{ operations | _calc_memory(memory) }}G
```

Comment:
Can you verify if this is the correct syntax? I found two references online for BSUB memory requests, and both of them state that this should be given as an integer value in _kilobytes_.

https://scicomp.ethz.ch/public/manual/LSF/9.1.3/lsf_admin/index.htm?chap_interactive_jobs.html~main
http://www.glue.umd.edu/lsf-docs/man/bsub.html

Reply:
Submissions that do not define a memory directive should not add this line, so this needs to be set in a conditional block. See the section below with `{% if walltime %}` for an example.

Reply:
I saw this documentation which says that we can use `G` or `GB` for `gigabytes` but by default, the memory is specified in kilobytes.
https://www.ibm.com/support/knowledgecenter/SSWRJV_10.1.0/lsf_command_ref/bsub.__m.1.html

Reply:
Excellent! That's the right reference, I just couldn't find it when I was googling. Let's use `GB` here.
```suggestion
#BSUB -M {{ operations | _calc_memory(memory) }}GB
```